cmd/aws-iam-authenticator/root.go

```diff
@@ -102,6 +102,7 @@ func getConfig() (config.Config, error) {
 		EC2DescribeInstancesQps:           viper.GetInt("server.ec2DescribeInstancesQps"),
 		EC2DescribeInstancesBurst:         viper.GetInt("server.ec2DescribeInstancesBurst"),
 		ScrubbedAWSAccounts:               viper.GetStringSlice("server.scrubbedAccounts"),
+		DynamicFilePath:                   viper.GetString("server.dynamicfilepath"),
```

Comment:
double checking, is the key supposed to be all lower case?

Reply:
checked above code, actually there is no fixed style here.                
                Address:                           viper.GetString("server.address"),
		Kubeconfig:                     viper.GetString("server.kubeconfig"),
		Master:                            viper.GetString("server.master"),